tests/extract.luau

```diff
@@ -45,26 +44,37 @@ return function(test: typeof(frktest.test))
 						end
 
 						-- Extract with unzip for comparison
-						local unzipOutput = process.spawn(`unzip`, { "-p", zipPath, entry:getPath() })
+						-- Fix pipe buffer overload by writing data into temp file and reading the temp file
+						local tmpOut = "/tmp/unzip_meta_"
+							.. tostring(os.time())
+							.. "_"
+							.. tostring(math.random(1, 1_000_000_000))
+							.. ".txt"
+
+						local command = `unzip -p {zipPath} {entry:getPath()} > {tmpOut}`
+						local execResult = process.exec("sh", { "-lc", command })
```

Comment:
Again, the `-p` file is to write to stdout, we should invoke the `unzip` command with different arguments to have it write to the file path directly, instead of piping its stdout using `sh` to a file descriptor.